code-examples/protect-page-login/nextjs/README.md

```diff
@@ -1,54 +1,53 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with
-[`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+This is a simple example application using @ory/elements-react & Next.js app
```

Comment:
Does it really make sense to document this here again? Would it not be easier to just link to the [docs](https://docs-n8rzswh56-ory.vercel.app/docs/getting-started/integrate-auth/nextjs)?